packages/kbn-apm-config-loader/src/utils/read_config.test.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { relative, resolve } from 'path';
+import { getConfigFromFiles } from './read_config';
+
+const fixtureFile = (name: string) => resolve(`${__dirname}/../../__fixtures__/${name}`);
```

Comment:
I think you may need to use this form for this test to work on Windows?

```suggestion
const fixtureFile = (name: string) => resolve(__dirname, '..', '..', '__fixtures__', 'name');
```